amplify/backend/api/collectionarchives/resolvers/Query.searchCollections.req.vtl

```diff
@@ -1,11 +1,11 @@
-#set( $indexPath = "/collection/doc/_search" )
+#set( $indexPath = "/collection/_search" )
 #set( $nonKeywordFields = ["visibility", "start_date"] )
 #if( $util.isNullOrEmpty($context.args.sort) )
-  #set( $sortDirection = "desc" )
-  #set( $sortField = "id" )
+  #set( $sortDirection = "asc" )
+  #set( $sortField = "title" )
```

Comment:
Why change to title?

Reply:
searchCollections began sorting by ID by default after the update causing tests to fail. I'm still working on figuring out why the query is no longer specifying the sort correctly. I'm working on that now but the default behavior is to sort by title and I can't think of any situation where we would want to sort by ID so it made sense to me for the fallback to be title.